### ISSUE TITLE
Document manual configuration

### ISSUE THREAD

**Issue by shlevy** [cachix/docs.cachix.org#32]
On my system cachix can't modify my configuration.nix or nix.conf. Would be nice to have documentation on what to configure manually.

**Comment by domenkozar** [comment]
@shlevy sure! Would something like `cachix use --mode manual` outputing steps work?

**Comment by shlevy** [comment]
Yeah that'd do it, especially if an error on `cachix use` recommended that
